libraries/commerce/product/actions/getProducts.js

```diff
@@ -65,7 +65,7 @@ const getProducts = ({
     const state = getState();
     const { offset, limit, ...hashParams } = params;
 
-    const sort = getSortOrder(state);
+    const { sort = DEFAULT_SORT } = params;
```

Comment:
You have params destructuring already in place two lines above ;) Please merge them.

Reply:
Changed somewhat